packages/ui/Scrollable/Scrollable.js

```diff
@@ -648,35 +653,34 @@ class ScrollableBase extends Component {
 
 		if (vertical) {
 			this.accumulatedTargetY += delta;
-			this.updateOverscrollEffect('vertical', this.accumulatedTargetY, overscrollTypes.scrolling, 1);
 		} else {
 			this.accumulatedTargetX += delta;
-			this.updateOverscrollEffect('horizontal', this.accumulatedTargetX, overscrollTypes.scrolling, 1);
 		}
 
-		this.start({
-			targetX: this.accumulatedTargetX,
-			targetY: this.accumulatedTargetY,
-			animate: true
-		});
+		this.start({targetX: this.accumulatedTargetX, targetY: this.accumulatedTargetY});
 	}
 
 	// overscroll effect
 
-	getScrollabilities = (orientation) => {
-		const bounds = this.getScrollBounds();
-		return (orientation === 'vertical') ? this.canScrollVertically(bounds) : this.canScrollHorizontally(bounds);
+	isOverEdge = (position, maxPosition) => {
```

Comment:
`get` function would be better than `is`.

Reply:
What about `getEdgeFromPosition`?

Reply:
👍